Jellyfin.Server/Program.cs

```diff
@@ -187,6 +187,30 @@ private static ServerApplicationPaths CreateApplicationPaths(StartupOptions opti
                 Directory.CreateDirectory(configDir);
             }
 
+            string cacheDir = Environment.GetEnvironmentVariable("JELLYFIN_CACHE_DIR");
+            if (string.IsNullOrEmpty(cacheDir))
+            {
+                if (options.ContainsOption("-cachedir"))
+                {
+                    cacheDir = options.GetOption("-cachedir");
+                }
+                else if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                {
+                    // $XDG_CACHE_HOME defines the base directory relative to which user specific non-essential data files should be stored.
+                    cacheDir = Environment.GetEnvironmentVariable("XDG_CACHE_HOME");
+                    // If $XDG_CACHE_HOME is either not set or empty, $HOME/.cache should be used.
+                    if (string.IsNullOrEmpty(cacheDir))
+                    {
+                        cacheDir = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), ".cache", "jellyfin");
+                    }
+                }
```

Comment:
No else{} for when running on windows.  I guess in that case we use the code in BaseApplicationPath CachePath 'get' to create the path to <datadir>/cache (yuck for me).  Might it be clear if that logic was moved here now, then getter becomes cleaner?

Reply:
Not a good idea, `<datadir>/cache` is the default for all implementations now, moving it out will force each separate implementation (ex: Mac app) to properly set up all paths.